server/db/order.js

```diff
@@ -0,0 +1,21 @@
+const Sequelize = require ('sequelize');
+const db = require('./_db');
+//import join here
+
+const Order = db.define('order', {
+  date: {
+    type: Sequelize.DATEONLY
+  }, 
+  totalCost: {
+    type: Sequelize.INTEGER
+  }, 
+  status: {
+    type: Sequelize.ENUM,
+    values: ['pending', 'completed']
```

Comment:
nit: same tense for values: 'pending', 'complete'

Reply:
Got it!